packages/components/src/filters/index.js

```diff
@@ -6,6 +6,7 @@ import { __ } from '@wordpress/i18n';
 import { Component, Fragment } from '@wordpress/element';
 import { find } from 'lodash';
 import PropTypes from 'prop-types';
+import moment from 'moment';
```

Comment:
Same thing here, avoid creating a dependency on `moment` for `@woocommerce/components`.